src/components/interface/AudioPlay.jsx

```diff
@@ -1,26 +1,82 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
+
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
 import styles from '../styles';
+import Button from '../interface/Button';
+import * as firebase from 'firebase';
 
 class AudioPlayer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      disabled: false
+      audioFile: undefined,
+      metadataLoaded: false
     };
   }
 
-  // TODO: handle audio playing file
+  loadAudio = () => {
+    this.pathRef.getDownloadURL().then(function (url) {
+      var xhr = new XMLHttpRequest();
```

Comment:
So it looks like instead of XMLHttpRequest it *may* be possible to use Fetch instead.

Here's a blog post about it: https://medium.com/beginners-guide-to-mobile-web-development/the-fetch-api-2c962591f5c